src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -351,6 +351,7 @@ def evaluate(
     # Target Run
     pf_client = PFClient(
         config={"trace.destination": trace_destination} if trace_destination else None,
+        skip_config_validation=True,

```

Comment:
What does this do ?

Reply:
And why do we need to skip validation ?

Reply:
It is used to avoid running validation inside of promptflow-devkit [here](https://github.com/microsoft/promptflow/blob/745704a5b7f868c61c71f7a12eb13ef695ab4333/src/promptflow-devkit/promptflow/_sdk/_tracing.py#L106C13-L106C18), which will result in loading of MLClient and promptflow-azure.

Reply:
Added our own implementation of a validation.